lib/index.js

```diff
@@ -2,18 +2,10 @@
 
 const requireIndex = require('requireindex');
 
-const gjs = require('./preprocessors/glimmer');
-
 module.exports = {
   rules: requireIndex(`${__dirname}/rules`),
   configs: requireIndex(`${__dirname}/config`),
   utils: {
     ember: require('./utils/ember'),
   },
-  processors: {
```

Comment:
we still probably want to export the parser -- @chancancode had a use case where the (now old) parser breaks on old code -- it'd be good for us to provide a way for folks to configure their own overrides in the lint config and set the parser:
```cjs
overrides: [
  {
    files: ['modern/**/*.gts'],
    parser: require('eslint-plugin-ember/gts-parser'),
    // ...
  }
]
```

Reply:
(we'd need to use `package.json#exports` to expose this tho -- maybe for another PR)

Reply:
Or rather have an option to specify the internal base parser. 
Now I'm just choosing between Babel parser and typescript eslint parser. But that could also be an option passed to the gts parser